client/my-sites/email/email-providers-in-depth-comparison/comparison-table/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { FunctionComponent } from 'react';
+import type { TranslateResult } from 'i18n-calypso';
+
+export type ProviderComparison = {
+	header: string;
+	tools: TranslateResult;
+	storage: TranslateResult;
+	importing: TranslateResult;
+	support: TranslateResult;
+	selectCallback: () => void;
+};
+
+type InDepthComparisonProps = {
```

Comment:
This should probably be named `ComparisonTableProps` to match the name of the component.